package.json

```diff
@@ -25,17 +26,19 @@
         "htmllint": "./node_modules/.bin/htmlhint --config .htmlhintrc.json src",
         "eslint": "./node_modules/.bin/eslint src",
         "tests": "mocha --require babel-register tests/test.js",
-        "cover": "node_modules/.bin/babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- tests/test.js"
+        "cover": "node_modules/.bin/babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- tests/test.js",
+        "eStart": "electron ."
     },
     "dependencies": {
         "@uirouter/angularjs": "1.0.15",
-        "angular": "1.6.9",
+        "angular": "^1.6.9",
         "angular-animate": "1.6.9",
         "angular-aria": "1.6.9",
         "angular-route": "1.6.9",
         "d3": "4.13.0",
         "font-awesome": "4.7.0",
         "jquery": "3.3.1",
+        "node": "^9.9.0",
```

Comment:
This is unnecessary, near the top of the file, we list "node": "8.6.2" under engines. We don't actually want to package node into our node_modules do we? I could be wrong about electron needing node like this but it kind of feels weird.